atlasdb-commons/build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     compile project(":commons-executors")
     compile 'javax.ws.rs:javax.ws.rs-api:2.0.1'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-annotations'
+    compile group: 'com.codahale.metrics', name: 'metrics-core'
```

Comment:
This is going to cause conflicts with folks that depend on newer versions of DW metrics via 'io.dropwizard.metrics:metrics-core'